Automattic/jetpack
# Carousel: Update to manage new core gallery block format

Fixes #21097

#### Changes proposed in this Pull Request:

* Upates handling of new core gallery block in the carousel as the selectors were changed in some of the core block iterations

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Check out this PR and run in local dev env along with latest release of gutenberg plugin with the experimental gallery flag enabled and the carousel lightbox option enabled
* Add a new gallery block with a number of images and check that the carousel loads correctly in the frontend
* Try changing the block to a few different alignments and check that the carousel continues to work
* Also test a number of galleries on the same post and page and that carousel runs for each
* Disable the gallery experiment flag and add another old style gallery block to the same page/post and make sure carousel runs for this also
